### PR TITLE
Adjusts Cook Book

### PR DESCRIPTION
**Pull request by LightFire53**
## What Does This PR Do
Brings the chef's cook book sprite more in line with the rest of the book/manual sprites.

## Why It's Good For The Game
 Consistency is nice, the chef's old cook book was a completely different shape, and it's cover was stylistically not great.

The old sprite was left in the code, I've just added a new icon, and changed the Chef's Recipe icon state to the new sprite.

## Images of changes

image

New sprite on the left, old one on the right.

image

## Changelog
:cl:LightFire53
tweak: Adjusts the chef's cook book sprite to bring more in line with the rest of the books in the game.
/:cl:

Baby's second PR, let see if I can update a sprite.

### CODE REVIEW

**Comment by MRgaty** [comment]
Yummy

**Comment by Fox-McCloud** [comment]
My concern here is. What is this, at a glance?

A spoon? A drumstick? A piece of candy? Typically you want something to be eye-catching and you know instantly what it is, without having to think about it.

**Comment by LightFire53** [comment]
> My concern here is. What is this, at a glance?
> 
> A spoon? A drumstick? A piece of candy? Typically you want something to be eye-catching and you know instantly what it is, without having to think about it.

A drumstick would be correct.

**Comment by LightFire53** [comment]
Sprite was updated with input from @FalseIncarnate. Brings more cook book vibes to it. Last sprite update unprompted, I promise!

**Review by Kyep — APPROVED** [review]

File: code/game/objects/items/weapons/manuals.dm [hunk 1]
```diff
@@ -838,7 +838,7 @@
 
 /obj/item/book/manual/chef_recipes
     name = "Chef Recipes"
-    icon_state = "cooked_book"
+    icon_state = "CookBook"
```
Comment (dearmochi) [inline code/game/objects/items/weapons/manuals.dm:841]: Not 100% sure what the convention on icon names is, but 99% of the icons have a `snake_case` name, so I'd prefer if that consistency was kept.

Comment (LightFire53) [reply code/game/objects/items/weapons/manuals.dm]: updated to cook_book

**Review by dearmochi — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**